discover/index.html

```diff
@@ -58,41 +58,41 @@ <h3 class="text-left" id="connect"><i class="fab fa-fw fa-mixcloud"></i> Get Con
 <p>If you are looking for <strong>imagery downloads</strong>, navigate to the <a href="{{ "/data/aerial-photography/" | prepend:site.baseurl }}">aerial photography</a> section of the data pages.</p>
 <h3 class="text-left" id="services"><i class="far fa-fw fa-map"></i> Available Services</h3>
 <ul class="dotless no-padding">
-  <li><i class="fas fa-fw fa-lock"></i> <code>Utah</code> – Combination of the most recent 1-meter NAIP and 6-inch Google imagery with scale dependent rendering <strong>(licensed)</strong>
```

Comment:
Define NAIP?

Reply:
naip is national agriculture imagery program

Reply:
Got it! Do we want to do the "appear when the mouse hovers over it" thing like some of the other acronyms? Or should we just spell it out?

Reply:
we certainly can. it's accomplished by <abbr title="expansion">ACRONYM</abbr>